style.css

```diff
@@ -90,3 +96,576 @@ p {
   display: flex;
   justify-content: space-between;
 }
+
+.works {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  margin: 80px auto;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.grid-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 5px;
+  background-color: #fff;
+  width: 327px;
+  height: 552px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.w-image1 {
+  padding-top: 15px;
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.toni {
+  width: 295px;
+  height: 36px;
```

Comment:
- [ ] Kindly make sure your page is not overlapping
You design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193102139-171ec17a-9e41-44b0-9eda-6bce817ba354.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193102480-2c63f945-22ba-46ad-9814-4451af5139bc.png)